app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -29,47 +29,73 @@
+                                <SortButton @sortBy='private_projects' />
+                            </div>
+                        </th>
+                        <th>
+                            <div class="header-content">
+                                {{t 'institutions.dashboard.users_list.public_registration_count'}}
+                                <SortButton @sortBy='public_registration_count' />
+                            </div>
                         </th>
-                        <th local-class='nested-header text-center' data-test-header-private-projects>
-                            <img src='/assets/images/global/closed-lock.svg'
-                                alt={{t 'institutions.dashboard.users_list.private_projects'}}
-                            >
+                        <th>
+                            <div class="header-content">
+                                {{t 'institutions.dashboard.users_list.private_registration_count'}}
+                                <SortButton @sortBy='embargoed_registration_count' />
+                            </div>
+                        </th>
+                        <th>
+                            <div class="header-content">
+                                {{t 'institutions.dashboard.users_list.published_preprint_count'}}
+                                <SortButton @sortBy='preprint' />
+                            </div>
                         </th>
                     </tr>
+
                 {{/let}}
             </list.header>
             <list.item local-class='item' as |institutionalUser|>
                 {{#if institutionalUser}}
                     <td data-test-item-name>
                         <OsfLink @href={{concat '/' institutionalUser.userGuid '/'}}>
-                            {{institutionalUser.userName}} ({{institutionalUser.userGuid}})
+                            {{institutionalUser.userName}}
                         </OsfLink>
                     </td>
                     <td data-test-item-department>{{institutionalUser.department}}</td>
-                    <td local-class='text-center' data-test-item-public-projects>{{institutionalUser.publicProjects}}</td>
-                    <td local-class='text-center' data-test-item-private-projects>{{institutionalUser.privateProjects}}</td>
-                {{else}}
-                    <td>{{placeholder.text lines=1}}</td>
-                    <td>{{placeholder.text lines=1}}</td>
-                    <td>{{placeholder.text lines=1}}</td>
-                    <td>{{placeholder.text lines=1}}</td>
+                    <td data-test-item-user-guid>
+                        <OsfLink @href={{concat '/' institutionalUser.userGuid '/'}}>
+                            {{institutionalUser.userGuid}}
+                        </OsfLink>
+                    </td>
+                    <td local-class='center-text' data-test-item-public-projects>{{institutionalUser.publicProjects}}</td>
```

Comment:
I see that the local-class name was `text-center` and is now `center-text` here, but hasn't been updated in the styles.scss file. It could be that we don't need this style anymore